frontend/src/shared/helpers/routing/reverse.ts

```diff
@@ -1,5 +1,24 @@
 import { Path, paths } from '@shared/consts/routing';
+import { UserActivationParams } from '@users/types/activation';
+import { isString, replace } from 'lodash';
 
-export const reverse = (path: Path): string => {
-  return paths[path];
+interface DetailedPath<TPath extends string, TParams extends Record<string, any>> {
+  path: TPath;
+  params: TParams;
+}
+
+type RegularPaths = Exclude<Path, 'users.activation'>;
+
+type ReversiblePath = RegularPaths | DetailedPath<'users.activation', UserActivationParams>;
+
+export const reverse = (path: ReversiblePath): string => {
+  if (isString(path)) return paths[path as Path];
+  else {
+    const pathWithParams = replace(
```

Comment:
To powinno byc bardziej generyczne, bo teraz dla kazdego nowego url bedzie tu trzeba dodawac logike podmianki. Z czasem ten plik bedzie mial 1000 linijek. Lepiej bedzie jak na przyklad przeiterujesz po paramsach i podmienisz kazda wartosc paramsa z placeholderem ze sciezki